components/scream/src/physics/p3/p3_functions_math_impl.hpp

```diff
@@ -1,110 +0,0 @@
-#ifndef P3_FUNCTIONS_MATH_IMPL_HPP
```

Comment:
Same for this file, it should be renamed and change `p3_functions_math*` to `p3_functions_saturation*`